examples/example_glfw_vulkan/CMakeLists.txt

```diff
@@ -15,14 +15,30 @@ set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DVK_PROTOTYPES")
 set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DVK_PROTOTYPES")
 
 # GLFW
-set(GLFW_DIR ../../../glfw) # Set this to point to an up-to-date GLFW repo
```

Comment:
The CMakelists.txt is already in imgui repo but this line is clearly inadequate as it makes the user clone the GLFW repo in a specific location relative to example folder.